Dong/두큐합.py

```diff
@@ -0,0 +1,33 @@
+from collections import deque
+
+def solution(queue1, queue2):
+    answer = 0
+    queue1 = deque(queue1)
+    queue2 = deque(queue2)
+    limit = (len(queue1)) * 4
+    total1 = sum(queue1)
+    total2 = sum(queue2)
```

Comment:
```suggestion
    queue1, queue2 = deque(queue1), deque(queue2)
    limit = (len(queue1)) * 4
    total1, total2 = sum(queue1), sum(queue2)
```

위와 같이 파이썬의 특징을 활용하여 가독성을 높이고 코드 줄도 줄이는 변수 선언은 어떠할까요??

Reply:
오 감사합니다 좋은 것 같아요